app/assets/stylesheets/show.scss

```diff
@@ -16,3 +16,42 @@
     padding-right: 5px;
   }
 }
+
+.voting-title-container {
+  padding-top: 20px;
+  vertical-align: middle;
+}
+
+.voting {
+  float: left;
+  padding-right: 5px;
+  font-size: 25px;
+}
+
+.review-container {
+  float: left;
+}
+
+.review-title {
+  margin-top: 5px;
+}
+
+// .big {
+//   font-size: 50px;
+//   color: red;
+// }
+//
+// .small {
+//   font-size: 20px;
+// }
+
+.voted a{
+  color: red;
+  &:hover{
```

Comment:
Rule declaration should be preceded by an empty line<br>Opening curly brace `{` should be preceded by one space